ci/docker/runtime_functions.sh

```diff
@@ -924,6 +924,17 @@ nightly_straight_dope_python3_multi_gpu_tests() {
       test_notebooks_multi_gpu.py --nologcapture
 }
 
+# check commit for flaky tests
+flaky_check_select_tests(){
+    set -ex
+    tools/flaky_tests/test_selector.py -b HEAD~1 HEAD
+}
+flaky_check_run_flakiness_checker(){
+    set -ex
+    export PYTHONPATH=./python/
```

Comment:
Should this line be `export PYTHONPATH=/work/mxnet/python/` like here: https://github.com/apache/incubator-mxnet/pull/11991/files#diff-1335fbaf3930b1438d9be18edb07a1a6R921 ?

Reply:
not sure, I was looking at the python2 unit test runtime function, which uses export PYTHONPATH=./python/ https://github.com/apache/incubator-mxnet/blob/64566872a28a9426f3ec20bcf0210ebb608854f8/ci/docker/runtime_functions.sh#L639

Reply:
But the pipeline is failing on not able to import mxnet here: http://jenkins.mxnet-ci-dev.amazon-ml.com/blue/organizations/jenkins/flaky-test-detector/detail/PR-11991/11/pipeline/67. Maybe we should switch to `/work/mxnet/python/`?

Reply:
Perhaps-- however, that run was triggered before the last commit, so I don't know which of these options we should go with. Perhaps @marcoabreu can help?

Reply:
@marcoabreu could you give some help here to get this to work?

Reply:
@cetsai @haojin2 you can test everything locally first, did it work for you?

Reply:
@haojin2 @cetsai @lebeg @marcoabreu requesting an update on this

Reply:
How can I help?

Reply:
@haojin2 I'm having trouble running this locally, can you try ?

Reply:
@haojin2 @cetsai Were you able to run this locally?

Reply:
Sorry, don't have any cycles for this at this moment...